domain/src/main/java/today/stepbeyond/examples/springbootexamples/domain/infrastructure/gateways/events/model/PetEvent.java

```diff
@@ -1,4 +1,4 @@
-package today.stepbeyond.examples.springbootexamples.infrastructure.gateways.events.model;
+package today.stepbeyond.examples.springbootexamples.domain.infrastructure.gateways.events.model;
```

Comment:
```suggestion
package today.stepbeyond.examples.springbootexamples.usecases.infrastructure.gateways.events.model;
```
In theory, there isn't any infrastructure objects contained domain layer.